connection.js

```diff
@@ -362,20 +354,12 @@ class Connection {
             let humanReadableMinmaxRank = rankToString(config[argNameString]);
 
             // then finally, the actual reject :
-            conn_log(`${user.username} ranking ${humanReadableUserRank} too ${lowHigh} ${rankedUnranked}: ${minMax} ${rankedUnranked}is ${humanReadableMinmaxRank}`);
-            return { reject: true, msg: `${minMax} rank ${rankedUnranked}is ${humanReadableMinmaxRank}, your rank is too ${lowHigh} ${rankedUnranked}` };
+            conn_log(`${user.username} ranking ${humanReadableUserRank} too ${lowHigh} ${rankedUnranked} : ${minMax} ${rankedUnranked} is ${humanReadableMinmaxRank}`);
```

Comment:
No need for spaces before `:`
It should read:
```
username ranking 2k too low for ranked games: min for ranked games is 1d.
```
not:
```
username ranking 2k too low for ranked games : min for ranked games is 1d.
```
Actually the whole sentence structure is a bit awkward, perhaps we can find a better format.

Also fancy idea: If a setting is the same for ranked/unranked, we could write
```
username ranking 2k too low to play: min rank to play is 1d.
```
bans already had that option. Again, we might want to find a sentence structure that's easy to build up.

Reply:
fixed the no space needed